system-x/services/azure/key-vault/src/main/java/software/tnb/azure/keyvault/account/KeyVaultAccount.java

```diff
@@ -0,0 +1,48 @@
+package software.tnb.azure.keyvault.account;
+
+import software.tnb.common.account.Account;
+import software.tnb.common.account.WithId;
+
+public class KeyVaultAccount implements Account, WithId {
+    private String vault_name;
```

Comment:
Why SnakeCase? Use CamelCase

Reply:
that's probably my bad, I started it in the credentials.yaml and then carried over to tnb so that we didn't need to specify which field was what in the yaml :face_with_head_bandage: it's like that in all services